packages/blade/src/components/TextArea/_decisions/decisions.md

```diff
@@ -0,0 +1,70 @@
+# TextArea
+
+A textarea field is an input field that allow users to input custom text entries which spans over multiple lines with a keyboard.
+
+<img src="./textarea-thumbnail.png" width="380" alt="" />
+
+## Design
+
+- [Figma - Alerts](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=11115%3A166804)
+
+## TextArea API
+
+**Sample Usage**
+
+```tsx
+<TextArea
+  label="Enter Message"
+  labelPosition="left"
+  placeholder="Enter your message"
+  showClearButton
+  helpText="Send message to author"
+  errorText="Message cannot be blank"
+  successText="Message sent"
+  validationState={textareaValue.length < 10 ? 'error' : 'success'}
```

Comment:
question: why this is not boolean instead of `string: 'error' | 'success'`?

Reply:
It might look like we have two states in first glance but we actually have 3 states. 

IDLE: 

<img width="490" alt="image" src="https://user-images.githubusercontent.com/35374649/188876631-869046c9-ba91-41b7-8327-77bf56a8f41e.png">

ERROR: 

<img width="491" alt="image" src="https://user-images.githubusercontent.com/35374649/188876678-756aef3e-ca35-4f31-a4c2-1270d2aeedd4.png">

SUCCESS: 

<img width="503" alt="image" src="https://user-images.githubusercontent.com/35374649/188876713-64a70236-3500-4759-ad75-44788ca3b79a.png">


So if we make the validationState into a boolean, then we will have to rely on the `undefined` value or the non-existence of the prop to map to the IDLE state. Which isn't very scalable. 

```ts
<Input validationState={isInvalid ? true : isValid ? true : undefiend} /> // <--- Weird ternaries
```